lib/rubocop/cop/style/each_for_simple_loop.rb

```diff
@@ -29,6 +25,21 @@ def on_block(node)
                                      'which iterates a fixed number of times.')
           end
         end
+
+        def autocorrect(node)
+          lambda do |corrector|
+            corrector.replace(node.children.first.source_range,
+                              "#{range_limit(node)}.times")
+          end
+        end
+
+        def_node_matcher :bad_each_loop, <<-PATTERN
```

Comment:
These 2 "node matcher" methods can be combined. The fact that one of them returns an integer doesn't stop it from also being used as a predicate.